Banking_App/src/main/java/com/cognologix/assignment/BankSystemAssignment/payloads/ApiResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.cognologix.assignment.BankSystemAssignment.payloads;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class ApiResponse {
```

Comment:
is this class getting used ?

Reply:
Yes, this is used in sending API response